tools/rebasehelpers/commitchecker/commitchecker.go

```diff
@@ -36,8 +36,13 @@ func main() {
 
 	errs := []string{}
 	for _, validate := range AllValidators {
-		err := validate(nonbumpCommits)
-		if err != nil {
+		if err := validate(nonbumpCommits); err != nil {
+			errs = append(errs, err.Error())
+		}
+	}
+	if len(os.Getenv("GODEP_RESTORE")) > 0 {
```

Comment:
We've definitely run into cases when refactoring that would have been extremely painful to make satisfy a strict bump verifier. 

 any time you are querying history for bumps and carry patches, you should really be doing it with git history of the actual sha referenced in the godep content, not based on commit titles… those are a convenience, not authoritative